test/avocado/testlib_avocado/seleniumlib.py

```diff
@@ -249,13 +255,18 @@ def wait(self, method, text, baseelement, overridetry, fatal, cond, jscheck):
     fatal - boolean if search is fatal or notice
     cond - use selenium conditions (aliases are defined above class)
     jscheck - use javascipt to wait for element has attribute-data loaded, it is safer, but slower
+    text_ - text to be present in element
```

Comment:
this seems very interesting, but little bit tricky, because this ``text_`` is used just in case ``cond`` set to ``text_in``, otherwise it is ignored.
But I have no better idea right now how to solve it. maybe something like using two conditions if possible, like: 
``until(cond((method, text)).until(text_in((method, text), text_))`` but I have no idea if it works like it.

Also this could lead that every method could support it eg. ``wait_id`` could forward it  to ``wait`` method, not just ``wait_in_text``, then there will be enough to just add ``wait_css`` method and remove ``wait_in_text``.

That's true, that your solution is more straight forward that just ``wait_in_text`` supports this option, but I don't see any reason to not support it for every method.

Reply:
I suppose `until(cond((method, text)).until(text_in((method, text), text_))` won't work, as `until()` will return a web element which doesn't have the `until()` method.

 I'd add `text_=None` to wait_id(), wait_link(), and wait_css(), and drop the wait_in_text()